src/keys/drivers/YubiKey.h

```diff
@@ -87,7 +94,11 @@ class YubiKey : public QObject
     bool m_initialized = false;
     QString m_error;
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
+    static QRecursiveMutex s_interfaceMutex;
+#else
     static QMutex s_interfaceMutex;
+#endif
```

Comment:
I don't even think we want a recursive mutex here anyway. This mutex is supposed to protect against multiple overlapping calls to find keys. These functions should not be recursively called in any normal context (and they aren't anyway).

TLDR - just change this to a standard QMutex

Reply:
Ok, done.